lib/javascripts/base64.js

```diff
@@ -0,0 +1,80 @@
+// This code was written by Tyler Akins and has been placed in the
+// public domain.  It would be nice if you left this header intact.
+// Base64 code from Tyler Akins -- http://rumkin.com
+
+module.exports = (function () {
+    var keyStr = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
+
+    var obj = {
+        /**
+         * Encodes a string in base64
+         * @param {String} input The string to encode in base64.
+         */
+        encode: function (input) {
```

Comment:
This is what was in Lotus, but we can probably use the [native functions](https://developer.mozilla.org/en/docs/Web/API/WindowBase64/Base64_encoding_and_decoding) `btoa` and `atob` now.